caddyhttp/proxy/reverseproxy.go

```diff
@@ -185,6 +186,8 @@ func NewSingleHostReverseProxy(target *url.URL, without string, keepalive int) *
 		rp.Transport = &http.Transport{
 			Dial: socketDial(target.String()),
 		}
+	} else if target.Scheme == "quic" {
+		rp.Transport = &h2quic.QuicRoundTripper{}
```

Comment:
Yeah, I meant that large block of logic. :) Okay, if they're not even available with the QuicRoundTripper then I wouldn't worry about it for now.